app/src/nvm/nvm_data.rs

```diff
@@ -0,0 +1,17 @@
+pub struct NVMData<T> {
```

Comment:
The ledger rust sdk does not define a `get_ref` method for nanos, and I recall the ledger developers mentioning plans to remove support for nanos from the rust sdk. So I added a definition of `NVMData` for nanos.